components/director/internal/open_resource_discovery/service_test.go

```diff
@@ -1058,6 +1101,81 @@ func TestService_SyncORDDocuments(t *testing.T) {
 				return client
 			},
 		},
+		// TODO: Delete the two tests below after the concept of central registry for Vendors fetching is productive
+		{
+			Name: "Success when resources are not in db and no Vendor is declared in Documents should Create them",
```

Comment:
"Success when resources are not in db and no `SAP` Vendor is declared in Documents should Create them"